src/VisualStudio/Core/Def/Implementation/FindReferences/ContainingTypeColumnDefinition.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.FindSymbols.Finders;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.FindUsages
```

Comment:
Should the namespace here match namespace for the base type, i.e. `Microsoft.VisualStudio.LanguageServices.Implementation.FindReferences`?